core/src/main/scala/akka/persistence/r2dbc/internal/R2dbcExecutor.scala

```diff
@@ -58,6 +58,7 @@ import org.slf4j.Logger
     }
 
     def asFutureDone(): Future[Done] = {
+      // FIXME really worth specific impl vs asFuture().map(_ => Done)(ExecutionContexts.parasitic)
```

Comment:
the difference is the "completed without first value". I think a publisher without any value is often used in the r2dbc api.

btw, I see that there is a bug in
```
if (promise.isCompleted)
```
it should be `!promise.isCompleted`